runtime/astar/src/lib.rs

```diff
@@ -646,16 +646,16 @@ pub const GAS_PER_SECOND: u64 = 40_000_000;
 
 /// Approximate ratio of the amount of Weight per Gas.
 /// u64 works for approximations because Weight is a very small unit compared to gas.
-pub const WEIGHT_PER_GAS: u64 = WEIGHT_PER_SECOND / GAS_PER_SECOND;
+pub const WEIGHT_PER_GAS: u64 = WEIGHT_PER_SECOND.ref_time() / GAS_PER_SECOND;
```

Comment:
Should'nt we use `saturated_div` here as well?

Reply:
I guess we can.

It had to be added to others because in those cases the type was `Weight`, not `u64`.